lib/driver.js

```diff
@@ -259,10 +259,19 @@ class XCUITestDriver extends BaseDriver {
     log.info(`${this.opts.bundleId} is in foreground`);
   }
 
-  async startWdaSession (app, bundleId) {
+  async startWdaSession (app, bundleId, processArguments) {
+    let appArguments = [];
```

Comment:
I think `args` would be better than `argsv`, since `argsv` makes it seem like command line arguments.